src/content/en/2021/markup.md

```diff
@@ -495,7 +495,7 @@ We've also looked at elements that didn't show up in the HTML or SVG spec, be it
   ) }}</figcaption>
 </figure>
 
-By far, the most popular one is <a hreflang="en" href="https://www.sliderrevolution.com/faq/developer-guide-output-class-tag-changes/">Slider Revolution</a>, with a majority of elements attributed to the framework. It more than tripled in popularity over the past year, which leads us to believe it might be a part of a popular template or site builder. A close second is <a hreflang="en" href="https://www.wix.com/">Wix</a>, the popular free site builder. We couldn't identify `pages-css`, but we'd love to hear any ideas for why the `pages-css` element is so popular, so let us know by [suggesting an edit](#explore-results) on GitHub.
+By far, the most popular one is <a hreflang="en" href="https://www.sliderrevolution.com/faq/developer-guide-output-class-tag-changes/">Slider Revolution</a>, with a majority of elements attributed to the framework. It more than tripled in popularity over the past year, which leads us to believe it might be a part of a popular template or site builder. A close second is <a hreflang="en" href="https://www.wix.com/">Wix</a>, the popular free site builder. We initially couldn't identify `pages-css`, but <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='alonko') }}">Alon</a> reached out and identified it as another custom element used by Wix, which also explains the similar page count to `wix-image`.
```

Comment:
```suggestion
By far, the most popular one is <a hreflang="en" href="https://www.sliderrevolution.com/faq/developer-guide-output-class-tag-changes/">Slider Revolution</a>, with a majority of elements attributed to the framework. It more than tripled in popularity over the past year, which leads us to believe it might be a part of a popular template or site builder. A close second is <a hreflang="en" href="https://www.wix.com/">Wix</a>, the popular free site builder. We initially couldn't identify `pages-css`, but [Alon Kochba](./contributors#alonko) reached out and identified it as another custom element used by Wix, which also explains the similar page count to `wix-image`.
```